core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -54,11 +54,15 @@ object ConsumerPerformance {
     val totalBytesRead = new AtomicLong(0)
     val consumerTimeout = new AtomicBoolean(false)
     var metrics: mutable.Map[MetricName, _ <: Metric] = null
+    val joinGroupTimeInMs = new AtomicLong(0)
+    val fetchTimeInMs = new AtomicLong(0)
 
     if (!config.hideHeader) {
-      if (!config.showDetailedStats)
-        println("start.time, end.time, data.consumed.in.MB, MB.sec, data.consumed.in.nMsg, nMsg.sec")
-      else
+      if (!config.showDetailedStats) {
```

Comment:
It's a little odd that we are reporting more information if the detailed option is not set, right? Perhaps we should at least report the additional join statistics down both paths?